src/controllers/tripController.js

```diff
@@ -216,3 +216,26 @@ export const searchTripRequests = async (req, res) => {
     return respondWithWarning(res, statusCode.internalServerError, 'Oops something bad happened');
   }
 };
+
+export const updateTripRequest = async (req, res) => {
+  const { params, auth, body} = req;
+  try { 
+    const [, tripRequest] = await editTripRequest(body, params.tripId, auth.id);
+    const payload = {
+    type: 'Trip Request edited',
```

Comment:
why are we making the type to be a `sentence` as opposed to `word`. This is ambiguous and it'll make sense to use `updateTrip` or `tripUpdated`, depending on which you find more appropriate.